docs/builds/guides/frameworks/vuejs.md

```diff
@@ -0,0 +1,645 @@
+	}
+};
+```
+
+### Creating a custom editor build
+
+Having configured `vue.config.js`, you can finally define your custom editor build. Install the packages the build will be made from:
+
+```bash
+npm install --save \
+	@ckeditor/ckeditor5-editor-classic \
+	@ckeditor/ckeditor5-essentials \
+	@ckeditor/ckeditor5-basic-styles \
+	@ckeditor/ckeditor5-basic-styles \
+	@ckeditor/ckeditor5-link \
+	@ckeditor/ckeditor5-paragraph
+```
+
+You can use more packages, depending on which feature are needed in your application.
+
+Create a separate `ckeditor.js` file that will combine editor features (`BoldPlugin`, `ItalicPlugin`, etc.) with the base editor (`ClassicEditorBase`) and export it as your custom build (`MyEditorBuild`):
+
+<info-box>
+	You can use the same methodology to integrate multiple CKEditor builds in your application as an optimized {@link builds/guides/integration/advanced-setup#scenario-3-using-two-different-editors "super build"}, e.g. when you need both `ClassicEditor` and `BalloonEditor`.
+</info-box>
+
+```js
+import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import EssentialsPlugin from '@ckeditor/ckeditor5-essentials/src/essentials';
+import BoldPlugin from '@ckeditor/ckeditor5-basic-styles/src/bold';
+import ItalicPlugin from '@ckeditor/ckeditor5-basic-styles/src/italic';
+import LinkPlugin from '@ckeditor/ckeditor5-link/src/link';
+import ParagraphPlugin from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+
+export default class MyEditorBuild extends ClassicEditorBase {}
+
+MyEditorBuild.builtinPlugins = [
+    EssentialsPlugin,
+    BoldPlugin,
+    ItalicPlugin,
+    LinkPlugin,
+    ParagraphPlugin
+];
+
+MyEditorBuild.defaultConfig = {
+    toolbar: {
+        items: [
+            'bold',
+            'italic',
+            'link',
+            'undo',
+            'redo'
+        ]
+    },
+    language: 'en'
+};
+```
+
+### Using the custom editor build
+
+Make sure your build is [available to the component](#editors) (e.g. in `main.js`):
+
+```js
+import CKEditor from '@ckeditor/ckeditor5-vue';
+import MyEditorBuild from './ckeditor';
+
+Vue.use( CKEditor, {
+	editors: {
+		myBuild: MyEditorBuild
+	}
+} );
+```
+
+<info-box>
+	You can always configure the component locally if you do not want to [specify editor builds](#local-build-configuration) at that point or just to [avoid using `Vue.use()`](#local-component-registration).
+</info-box>
+
+Now all you need to do is specify `myBuild` in the [`editor`](#editor) directive to get your CKEditor instance use `MyEditorBuild`:
+
+```html
+<template>
+	<div id="app">
+		<ckeditor editor="myBuild" v-model="editorData" :config="editorConfig"></ckeditor>
+	</div>
+</template>
+
+<script>
+	export default {
+		name: 'app',
+		data() {
+			return {
+				editorData: '<p>Content of the editor.</p>',
+				editorConfig: {
+					// Configuration of the editor.
+				}
+			};
+		}
+	}
+</script>
+```
+
+## Plugin configuration
```

Comment:
What's "plugin" here? Do you mean "`<ckeditor>` component"? If so, it'd be good to avoid using "plugin" because I thought that this is about CKEditor plugins.